### ISSUE TITLE
Improve visual clarity and consistency of the `send_button_html` and `download_list_html()` elements

### ISSUE THREAD

**Issue by nonprofittechy** [SuffolkLITLab/docassemble-AssemblyLine#563]
The current buttons are a little messy, especially if you have two separate bundles:

![image](https://user-images.githubusercontent.com/7645641/188484722-d795ea85-dd04-4780-9491-194130ce2fe9.png)

The `download_list_html()` buttons and the `send_button_html()` buttons:

1. Are not on the same grid
2. Are not clearly separated or grouped by a proper amount of white space
3. Have a confusing visual hierarchy--the `h5` that labels the `send_button_html()` output is larger than the name of the document, and the document title is bold while the "send as email" heading is fairly light weight
4. Are not clearly grouped together as one logical unit (unifying the send button with the list of documents)

See also #562

**Comment by nonprofittechy** [comment]
@mnewsted would want to coordinate on any changes to this with you, since you usually want to avoid look and feel changes that are "upstream". Or let us know if you're fine with any changes to improve this

**Comment by plocket** [comment]
Here are some adjustments. They depart from past decisions by right-aligning the "view" and "download" buttons, leaving a big gutter between the document's titles and their buttons. The document titles are also right aligned here, which really makes that gutter stick out:

<img width="549" alt="final_page_changes" src="https://user-images.githubusercontent.com/52798256/188659582-c5a64d4c-5688-4c7f-837b-2d0d6a52c3b1.png">

I couldn't get the same final page as the one above, so I couldn't address item 2 in the concerns expressed above.

**Comment by nonprofittechy** [comment]
1. Get rid of the heading "get a copy in email" (just leave it to the developer to decide if editable or not)
2. Put the send button into the same container--an optional field output by `download_list_html()`
3. Add some more whitespace between bundles. And make the title of the bundle and optional thing to display.

**Comment by mnewsted** [comment]
Thanks for including me in this @nonprofittechy Sorry I didn't see it until today. I'm not sure how all of these changes will impact the current layout of our download screen (see below example). Maybe because we haven't broken down our forms on the download page into forms for personal use v. forms to be filed, the current layout (single download table, single email button) works for us. I do think the "Get a copy of the documents in email" could be removed, as long as we can add our own header/text, which I think we can since it's part of the subquestion.

![image](https://user-images.githubusercontent.com/66691956/189721970-3f9a97d0-a334-4b21-8d96-5348463750db.png)

**Comment by plocket** [comment]
> as long as we can add our own header/text

@mnewsted - your own header text for the bundle or your own header text for the email sending section?

**Comment by mnewsted** [comment]
@plocket I was referring to the email field/button header "Get a copy of documents in email" (or "Get a copy of your forms by email" in the ILAO interview). I think developers can include whatever text is appropriate above that field/button in the subquestion.

**Comment by plocket** [comment]
@mnewsted - if we align the "send" button to the right as shown above, would that still work for you?

**Comment by plocket** [comment]
New send email row
- [ ] add a new parameter to `download_list_html()` to include the send email button row
- [ ] No header, no "include editable"
- [ ] remove the big gutter between the buttons and the text

Current send email button
- [ ] remove the H4 and replace with H2 class="h4"
- [ ] Possibly right align the send button (after feedback from collaborators)

**Comment by nonprofittechy** [comment]
We talked about replacing the "download zip" button with two buttons of the same size as "view" and "download".

**Comment by mnewsted** [comment]
This is an interesting idea. Very interesting. I agree button size might need to be adjusted for consistency. Also, What was decided with the location & formatting of the send email field/button? Thanks!

**Comment by plocket** [comment]
I think the styles for the isolated email section (for the email field and the button), the widths and alignment are the only things changing and they're still being decided. As far as technical details go, we're turning that section into a `fieldset` and the current `h5` tag will become a `legend` tag, but the style of the heading will stay the same. I don't recall working with you on style adjustments there, but if yours need updating I can try to help with that, @mnewsted.

**Comment by plocket** [comment]
Da's fields are left aligned, so I couldn't implement that without messing up our right-hand alignment. I tried something else:

<img width="577" alt="flex_grow" src="https://user-images.githubusercontent.com/52798256/193860865-b1c6bab4-3bcb-4f50-a6fd-92c2355a55ce.png">

The table "send" button isn't the same width as the other buttons and generally the columns don't align with the ones above. The gap looks wider between the email field and the send button, but it's really the same as the other rows. I can line up those columns if we're ok with the email field being significantly smaller.

I'm not sure what we want to do with the formatting of the "send email" fields of the isolated email section. I made the button a little wider. Haven't worked on the HTML there, just showing what it looks like visually. I can work on having them align with the table fields better, but then I'll have to put them into bootstrap columns like the table.

Thoughts?
